### DISCUSSION TITLE
A container to have multiple child objects?

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [fabricjs/fabric.js#7733] [Q&A] [answered]
Hey there,

I need a "parent" object that has 3 child objects, and when the parent is updated the children would be too.

For example, when changing the parent's position, all the children also changed depending on the parent's position:

Normal position:
```
- Parent {left: 10, top: 100};
   - Child 1: {left: 0, top: -5} (local), global would have {left: 10, top: 95}
```

Parent change:
```
- Parent {left: 90, top: 55};
   - Child 1: {left: 0, top: -5} (local), global would have {left: 90, top: 50}
```

I've already made a code that does what I say above, but I think it's something very complex, I wanted something simpler and preferably native

Code: https://codesandbox.io/s/fabricjs-child-pxnp97

**Comment by ShaMan123** [comment, marked as answer]
#7670

**Reply by 201flaviosilva** [reply]
Thanks

**Comment by asturur** [comment]
isn't this the normal group?

**Reply by ShaMan123** [reply]
seems so, unless there's a need for re-layout

**Reply by 201flaviosilva** [reply]
Sorry if it's confusing, I'm not very fluent in English.

But my goal is to have sub-objects whose positions depend on a parent object, I think the simplest way to demonstrate this is in the case of Unity Engine. In Unity, we can nest objects, and when an object changes it also changes its children depending on its transformation properties.

From what I've seen, and I've been testing a group on the fabric, the rotation is done taking into account the coordinates in the middle of all objects :)

https://codepen.io/201flaviosilva/pen/mdqaZey

But maybe it's just that I'm being too noob
